File: javascript/Day_1/Question_10.js
year = 1999;

const leapYear = (y) => {
  if (y % 400 == 0) {
    return `${y} is leap year`;
  } else if (y % 100 == 0) {
    return `${y} is not leap year`;
  } else if (y % 4 == 0) {
    return `${y} is leap year`;
  } else {
    return `${y} is not leap year`;
  }
};

const result = leapYear(year);

console.log(result);

File: python/Day_1/Question_10.py
year = int(input('Year: '))

def leapYear(y):
    if(y % 400 == 0):
        return f"{y} is leap year"
    elif(y % 100 == 0):
        return f"{y} is not leap year"
    elif(y % 4 == 0):
        return f"{y} is leap year"
    else:
        return f"{y} is not leap year"

result = leapYear(year)

print(result)